UITests/TransferMethod/AddTransferMethodBankAccountBusinessTests.swift

```diff
@@ -7,28 +7,62 @@ class AddTransferMethodBankAccountBusinessTests: BaseTests {
     let bankAccount = NSPredicate(format: "label CONTAINS[c] 'Bank Account'")
 
     override func setUp() {
-        profileType = .business
         super.setUp()
 
-        setUpBankAccountScreen()
+        app = XCUIApplication()
+        app.launchEnvironment = [
+            "COUNTRY": "US",
+            "CURRENCY": "USD",
+            "ACCOUNT_TYPE": "BANK_ACCOUNT",
+            "PROFILE_TYPE": "BUSINESS"
+        ]
+        app.launch()
+
+        mockServer.setupStub(url: "/graphql",
+                             filename: "TransferMethodConfigurationBankAccountBusinessResponse",
+                             method: HTTPMethod.post)
+
+        app.tables.cells.staticTexts["Add Transfer Method"].tap()
+        spinner = app.activityIndicators["activityIndicator"]
+        waitForNonExistence(spinner)
+        addTransferMethod = AddTransferMethod(app: app, for: .bankAccount)
     }
 
     func testAddTransferMethodBankAccountBusiness_displaysElementsOnBusinessProfileTmcResponse() {
-        waitForNonExistence(spinner)
+        XCTAssert(app.navigationBars["Bank Account"].exists)
```

Comment:
@skoong @jsaini-hw 
Should we have similar requirements for all the tests?   
I mean adding `message` into all the `XCTAssert`'s, like    
```
XCTAssert(app.navigationBars["Bank Account"].exists, "The 'Bank Account' should exist")
```

Reply:
Yes, I think thats good to have.  Maybe in the future we can add it to the assertions.